geopolars/Cargo.toml

```diff
@@ -14,13 +14,30 @@ rust-version = "1.60"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
+default = [
+  "polars/fmt",
+  "polars/zip_with",
+  "polars/csv-file",
+  "polars/temporal",
+  "polars/fmt",
+  "polars/dtype-slim",
+]
+
+# basic-polars = ["polars/ipc", "polars/dtype-u8", "polars/dtype-i8", "polars/private"]
+
 proj = ["dep:proj"]
 
 [dependencies]
 geozero = { version = "0.9.4", features = ["with-wkb"] }
 rstar = "0.9.3"
 geo = { git = "https://github.com/georust/geo.git", rev = '578e213875915e1f895892487b5a36ca0d91fac3' }
-polars = { version = "0.22.8", features = ["ipc", "dtype-u8", "dtype-i8"] }
+# Only the _minimal_ features to get polars to compile should be declared here
+polars = { version = "=0.22.6", features = [
+  "dtype-u8",
+  "dtype-i8",
+  "ipc",
+  "private",
+], default-features = false }
```

Comment:
I think any features that are _required_ to get the parts of polars we use to compile should be defined here, then more features can be defined above.